### ISSUE TITLE
Update Release Notes template to use Protocol

### ISSUE THREAD

**Issue by stephaniehobson** [mozilla/bedrock#6823]
Update Release Notes template to use Protocol

Blocked on:
- ~Second nav in pages mozilla/protocol#300~

Nice to have:
- ~Label for button of second mobile nav mozilla/protocol#313~

WIP screenshot:
![screenshot_2019-02-08 firefox 65 0 see all new features updates and fixes](https://user-images.githubusercontent.com/854701/52757294-0c1a0d80-2fb9-11e9-9c8a-73fa28b1641f.png)
